modules/accessing-inference-endpoint-for-model-deployed-on-single-model-serving-platform.adoc

```diff
@@ -43,12 +43,26 @@ The inference endpoint for the model is shown in the *Inference endpoint* field.
 * `:443 fmaas.GenerationService/Generate`
 * `:443 fmaas.GenerationService/GenerateStream`
 +
-NOTE: To query the endpoint for the TGIS standalone runtime, you must also download the files in the `proto` directory of the IBM link:https://github.com/IBM/text-generation-inference[text-generation-inference^] repository.
+NOTE: To query the endpoint for the TGIS standalone runtime, you must also download the files in the `proto` directory of the Open Data Hub link:https://github.com/opendatahub-io/text-generation-inference/blob/main/proto[text-generation-inference^] repository.
 
 *OpenVINO Model Server*
 
 * `/v2/models/<model-name>/infer`
 
+*vLLM ServingRuntime for KServe*
+
+* `:443/version`
+* `:443/docs`
+* `:443/v1/models`
+* `:443/v1/chat/completions`
+* `:443/v1/completions`
+* `:443/v1/embeddings`
++
+NOTE: The vLLM runtime is compatible with the OpenAI REST API and TGIS gRPC API. If you have deployed a model using the TGIS Standalone runtime, you have the option to migrate to the vLLM runtime, if desired. To use the TGIS gRPC API to query the endpoints for a model deployed using the vLLM runtime, refer to the paths that are used for the TGIS Standalone runtime. For a list of models supported by the vLLM runtime, see link:https://docs.vllm.ai/en/latest/models/supported_models.html[Supported models].
```

Comment:
"If you have deployed a model **by** using..." (otherwise, it sounds like the model **is** using the TGIS Standalone runtime)